bedrock/firefox/tests/test_base.py

```diff
@@ -353,22 +355,22 @@ def test_fx_dev_browser_57_0_a2_whatsnew(self, render_mock):
         assert template == ["firefox/developer/whatsnew.html"]
 
     @override_settings(DEV=True)
-    @patch.dict(os.environ, SWITCH_DEV_WHATSNEW_68="False")
-    def test_fx_dev_browser_68_0_a2_whatsnew_off(self, render_mock):
+    @patch.dict(os.environ, SWITCH_FIREFOX_DEVELOPER_WHATSNEW_MDNPLUS="False")
+    def test_fx_dev_browser_100_0_a2_whatsnew_off(self, render_mock):
         """Should show regular dev browser whatsnew template"""
         req = self.rf.get("/en-US/firefox/whatsnew/")
-        self.view(req, version="68.0a2")
+        self.view(req, version="100.0a2")
         template = render_mock.call_args[0][1]
         assert template == ["firefox/developer/whatsnew.html"]
 
     @override_settings(DEV=True)
-    @patch.dict(os.environ, SWITCH_DEV_WHATSNEW_68="False")
-    def test_fx_dev_browser_100_0_a2_whatsnew_off(self, render_mock):
```

Comment:
Same thing here,;the 68 tests were redundant so the diff gets a little messy where old tests and new tests overlap.